lib/uri/ws.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: false
+# = uri/ws.rb
+#
+# Author:: Matt Muller <mamuller@amazon.com>
+# License:: You can redistribute it and/or modify it under the same term as Ruby.
+# Revision:: $Id$
```

Comment:
Can you remove this? This line is not required with Git.

Reply:
Sure, I can remove these.

I put them there because they were at the top of all the other files (such as [this](https://github.com/ruby/uri/blob/master/lib/uri/ftp.rb#L4)).. How else will my programming legacy live on?!